cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-deny-syntax.asciidoc

```diff
@@ -52,7 +40,8 @@ allows the specified entities to be found
 ** `READ +{props}+`
 +
 allows the specified properties to be read on the found entities.
-Note that if the way the entity is found requires reading the property, it will not be found.
+Note that granting property READ access does not imply that the entities with that property can be found.
+For example if there is also a DENY on it, the entity will not be found.
```

Comment:
Or missing TRAVERSE?

Reply:
Thats why i was saying "for example". I was not meaning to give a complete list, just an example